packages/datadog-plugin-jest/src/index.js

```diff
@@ -148,7 +148,19 @@ function createHandleTestEvent (tracer, testEnvironmentMetadata, instrumenter) {
         try {
           result = await specFunction()
           // it may have been set already if the test timed out
-          if (!testSpan._spanContext._tags['test.status']) {
+          let suppressedErrors = []
+          const context = environment.getVmContext()
+          if (context) {
+            suppressedErrors = context.expect.getState().suppressedErrors
+          }
+          if (suppressedErrors && suppressedErrors.length) {
+            const testError = suppressedErrors[0]
+            testSpan.setTag(ERROR_TYPE, testError.constructor ? testError.constructor.name : 'Error')
```

Comment:
This should probably be handled by `setTag('error', error)`.

Reply:
I think we're in the same situation as in https://github.com/DataDog/dd-trace-js/pull/1298 with this comment: https://github.com/DataDog/dd-trace-js/pull/1298#discussion_r611666544

Reply:
Looks like this was kept for later, at least from my understanding of the last comment. Wouldn't now be a good time to do this since it seems to keep creeping back?

Reply:
sure. Let me give it a try 👍

Reply:
should be done in https://github.com/DataDog/dd-trace-js/pull/1404/commits/7adc56f6c6387ed47b786b4dfae1b05225369fcb